src/devices/Directory.Build.props

```diff
@@ -1,6 +1,8 @@
 <Project>
   <Import Project="../../eng/Compilers.props" />
   <Import Project="../../eng/Versions.props" />
+  <!-- Disabled until all projects are merged -->
```

Comment:
NIT: it would be good if we can log an issue abou tthis and have that added to this comment to be able to track progress.

Reply:
We have https://github.com/dotnet/iot/issues/816 I'll add a check box there

Reply:
If we could just add the pointer to that to this comment that would be awesome